README.md

```diff
@@ -47,7 +47,8 @@ Where `x.y.z` is the archetype version number that you want to use (see archetyp
 | packageGroup          |         | Content Package Group name         |
 | siteName              |         | AEM site name                      |
 | optionAemVersion      | 6.5.0   | Target AEM version                 |
-| optionIncludeExamples | y       | Include Component Library examples |
+| optionIncludeExamples | y       | Include sample content package     |
+| optionEmbedConnector  | n       | Embed CIF connector in all package |
 
 Note: If the archetype is executed in interactive mode the first time properties with default values can't be changed (see
```

Comment:
Should we add one more note here for the `optionEmbedConnector` option? Describing the behavior and what to do to create a one-includes-all sample package?